lightly_studio/src/lightly_studio/api/routes/api/frame.py

```diff
@@ -9,18 +9,16 @@
 
 from lightly_studio.api.routes.api.validators import Paginated, PaginatedWithCursor
 from lightly_studio.db_manager import SessionDep
-from lightly_studio.models.video import VideoFrameViewsWithCount
-from lightly_studio.resolvers import (
-    video_frame_resolver,
-)
+from lightly_studio.models.video import VideoFrameTable, VideoFrameView, VideoFrameViewsWithCount
+from lightly_studio.resolvers import video_frame_resolver
 from lightly_studio.resolvers.video_frame_resolver.get_all_by_dataset_id import VideoFramesWithCount
 
 frame_router = APIRouter(prefix="/datasets/{video_frame_dataset_id}/frame", tags=["frame"])
 
 
 @frame_router.get("/", response_model=VideoFrameViewsWithCount)
 def get_all_frames(
-    video_frame_dataset_id: Annotated[UUID, Path(title="Video frame dataset Id")],
+    video_frame_dataset_id: Annotated[UUID, Path(title="Dataset Id")],
```

Comment:
Minor: I think the previous was more descriptive. This string is displayed in openapi docs.